redisson/src/main/java/org/redisson/RedissonRateLimiter.java

```diff
@@ -242,6 +242,21 @@ public boolean trySetRate(RateType type, long rate, long rateInterval, RateInter
               + "return redis.call('hsetnx', KEYS[1], 'type', ARGV[3]);",
                 Collections.singletonList(getName()), rate, unit.toMillis(rateInterval), type.ordinal());
     }
+
+    @Override
+    public void setRate(RateType type, long rate, long rateInterval, RateIntervalUnit unit) {
+        setRateAsync(type, rate, rateInterval, unit);
+    }
+
+    @Override
+    public RFuture<Void> setRateAsync(RateType type, long rate, long rateInterval, RateIntervalUnit unit) {
+         return commandExecutor.evalWriteAsync(getName(), LongCodec.INSTANCE, RedisCommands.EVAL_BOOLEAN,
+                "redis.call('hset', KEYS[1], 'rate', ARGV[1]);"
+                        + "redis.call('hset', KEYS[1], 'interval', ARGV[2]);"
+                        + "redis.call('hset', KEYS[1], 'type', ARGV[3]);"
+                        + "return redis.call('del', KEYS[2], KEYS[3]);",
```

Comment:
we don't need to use return in case of void result

Reply:
@mrniko addressed